docs/progress-guarantees.md

```diff
@@ -61,37 +62,32 @@ class DataHolderTest {
 }
 ```
 
-TODO: I would suggest adding a "To sum up" subsection to each part, and adding some words on what we have learned along with the full code. A link to the next chapter can also be put there.
-
-> Get the full code [here](../src/jvm/test/org/jetbrains/kotlinx/lincheck/test/guide/DataHolderTest.kt).
-
 The test will fail with the following output:
 
 TODO: but an active lock has been DETECTED, let's make a PR
```

Comment:
I didn't get you, please explain what is wrong with this output?

Reply:
"an active lock has been detected" sounds more natural than "an active lock has been found". Anyway, let's forget about this.